openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -211,9 +211,6 @@ def run(self):
         fwd = problem._mode == 'fwd'
         optimizer = self.options['optimizer']
 
-        # Metadata Setup
-        self.metadata = create_local_meta(optimizer)
```

Comment:
Since this happens in record_iteration now, you can remove the `create_local_meta` from the imports.

Reply:
right... missed that, thanks.